core/src/main/java/hudson/util/ProcessTree.java

```diff
@@ -1979,9 +1979,6 @@ Object writeReplace() {
      * This feature involves some native code, so we are allowing the user to disable this
      * in case there's a fatal problem.
      *
-     * <p>
-     * This property supports two names for a compatibility reason.
      */
-    public static boolean enabled = !SystemProperties.getBoolean(ProcessTreeKiller.class.getName()+".disable")
-                                 && !SystemProperties.getBoolean(ProcessTree.class.getName()+".disable");
+    public static boolean enabled = !SystemProperties.getBoolean(ProcessTree.class.getName()+".disable");
```

Comment:
Would revert this. There's no real reason for this change; just type out the obsolete class name.

Reply:
Thanks for pointing that solution out. Changed it as requested.